src/lib/github/apiActions/apiActions.ts

```diff
@@ -0,0 +1,62 @@
+"use server";
+
+import {
+  BaseGithubApiActions,
+  GitHubPaginatedResponse,
+} from "./baseApiActions";
```

Comment:
It's better to use the path alias `'@/*'` than relative paths so that import paths don't change as a result of _this_ file changing location, if that makes sense?